config/locales/en.yml

```diff
@@ -177,6 +177,11 @@ en:
         unknown: Don't know
       language:
         English: English
+      practical_support:
+        companion: Companion
```

Comment:
@harumhelmy can you verify that these are good as starter, general categories of practical support?

Reply:
Thanks for gathering the feedback!